src/Microsoft.Azure.Batch.SoftwareEntitlement.Server/Model/IVerificationRequestBody.cs

```diff
@@ -0,0 +1,18 @@
+namespace Microsoft.Azure.Batch.SoftwareEntitlement.Server.Model
+{
+    /// <summary>
+    /// A request body which needs to be verified to request an entitlement.
+    /// </summary>
+    public interface IVerificationRequestBody
```

Comment:
What motivates this interface?

Reply:
There are a couple of different request body classes used for verifying a token (for approve and acquire), which I wanted to treat in the same way for the purposes of creating a `TokenVerificationRequest`. Could have used a base class, but prefer to avoid inheritance in DTOs.